main.go

```diff
@@ -324,16 +324,17 @@ func connectAction(ctx *cli.Context) error {
 		// Generate a new http client configured with mutual TLS certificate
 		tlsConfig, err := config.CreateTLSClientConfig()
 		if err != nil {
-			errorMessages[BrandName] = fmt.Errorf("cannot configure TLS: %w", err)
+			log.Debug("Cannot configure TLS: ", err)
```

Comment:
Do we really want to only log this output if the log level is greater than or equal to "debug"? This feels more like the sort of error we should log at the "error" level. Logging at the error level won't result in a change to the exit code, but it will ensure the error is reported at the default logging level.

Reply:
I wanted to discuss this :smile:. I wasn't sure if putting the log level to error was a good idea or if it was better to use the "warning" level. This error doesn't imply the `rhc` has failed but it couldn't gather some information. 

I'm imagining the case there is a temporary error in the network connection that makes this call fail. If this error is logged in the "error" level this will be printed in the output (if the configuration file is left by default). But it will not be printed if the "warning" level is set.

I'm not sure what behavior we want for this example.

Reply:
Error messages are only useful to users if they're actionable. Meaning, is there something the user can do in response to the error message to fix the problem. In this case, I doubt there is much they can do. Maybe we can log the TLS error and the HTTP client configuration error at a non-default log level (such as "warning"), and log a final error message if `getConfProfile` fails, at least informing the user that *something* that was supposed to work did not work.

Reply:
That seems reasonable